api/src/wfl/boot.clj

```diff
@@ -1,7 +1,8 @@
 (ns wfl.boot
```

Comment:
Could we call this `wfl.build` instead?

Reply:
I'll do that in a follow-up to maintain blame.

Reply:
Ooh good call!